tabs/package.json

```diff
@@ -20,6 +20,7 @@
     "@types/react": "^16.14.6",
     "@types/react-dom": "^16.9.12",
     "@types/react-router-dom": "^5.1.7",
+    "chokidar": "^3.5.3",
```

Comment:
what's this for?

Reply:
Without this, I was getting compilation issues on my Windows machine. I searched my issue and it seems it has something to do with npm and potentially may be Windows-only. Chokidar is a React dependency and npm was having trouble auto-resolving it due to versioning conflicts somewhere. Adding this line effectively made the dependency explicit and stopped the errors on my end.